### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Wednesday, 10th of April 2024 2:30pm UTC

### DISCUSSION THREAD

**Discussion by tormath1** [flatcar/Flatcar#1413] [Flatcar Office Hours]
In the office hours we discuss news and happenings in and around Flatcar, provide a space for our user community to discuss with maintainers, and do our release planning.

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  https://www.youtube.com/watch?v=isvqZIkahNo
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items

- A short Kubecon review (@tormath1: more cloud providers, talk recording links, @miao0miao: [TAG-Runtime talk](https://www.youtube.com/watch?v=8kXwAGZ8Iu0&ab_channel=CNCF%5BCloudNativeComputingFoundation%5D) and [SPOS panel](https://youtu.be/uD-T9LdfvrM))
- ZFS extension (@jepio)
- Hyper-V support improvements (@ader1990)
- Brightbox official support w/ demo (@andatche)

### Release Planning

- https://github.com/orgs/flatcar/projects/7/views/8 (@dongsupark)

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [x] Close this discussion as "Resolved".

**Comment by ader1990** [comment]
Hello, I would like to add to the agenda the Hyper-V support improvements (the ability to build a Hyper-V VHDX Flatcar image and some nice QoL improvements like docs, fixed ignition support, LUKS with TPM also tested and working).

**Reply by miao0miao** [reply]
Please go on and add the item :)

**Comment by andatche** [comment]
Hi! I'm planning to do a quick introduction to Brightbox (https://www.brightbox.com) and discuss the availability of official Flatcar images on Brightbox, then hopefully go through a quick demo of building a simple 3 node etcd cluster.

**Comment by tormath1** [comment]
Thanks everyone, the recording is available here: https://www.youtube.com/watch?v=isvqZIkahNo
